assets/script/script.js

```diff
@@ -96,3 +88,125 @@ function renderLeagueStats() {
     }
   }
 }
+
+// get QueryString parameters from welcome page and use it to render results
+const urlParams = new URLSearchParams(window.location.search);
+function loadTeamFromWelcomePage() {
+  for (let i = 0; i < 20; i++) {
```

Comment:
What is the significance of 20? It appears in a couple of places: maybe define a variable.